[test_parallel_sequential.py:802] | Sanity tests | ttnn-sanity-tests / ttnn fused group 1 [sim_blackhole]

## CI Failure Report

<!-- AUTO-TRIAGE-SUMMARY-START -->
**Summary:** [CI] TypeError in test_fp32_mismatch_error: expect_error() missing required positional argument 'message'
<!-- AUTO-TRIAGE-SUMMARY-END -->

**Workflow:** Sanity tests
**Job:** ttnn-sanity-tests / ttnn fused group 1 [sim_blackhole]

### Regression timeline

- **Last passing commit:** `25c2bfb980e8` — 2026-07-04T13:29:08Z — [run](https://github.com/tenstorrent/tt-metal/actions/runs/28707726397)
- **First failing commit:** `53fd14fd7f49` — 2026-07-05T13:31:45Z — [job](https://github.com/tenstorrent/tt-metal/actions/runs/28742470370/job/85231758532)
- **Most recent failing commit:** `33105d7568a8` — 2026-07-05T13:57:43Z — [job](https://github.com/tenstorrent/tt-metal/actions/runs/28743170285/job/85233693968)

### Failing job URLs (last 2 runs)

- https://github.com/tenstorrent/tt-metal/actions/runs/28743170285/job/85233693968
- https://github.com/tenstorrent/tt-metal/actions/runs/28742470370/job/85231758532

### Failing test path(s)

- `tests/ttnn/unit_tests/operations/fused/parallel_sequential/test_parallel_sequential.py::TestMatmulFusion::test_fp32_mismatch_error`

### Error excerpt (terminal failure from logs)

**Run 1** (`33105d7568a8`):
```
__________________ TestMatmulFusion.test_fp32_mismatch_error ___________________
    def test_fp32_mismatch_error(self, device, expect_error):
        """MM(fp32=off) + LN(fp32=on) should raise."""
        ...
>       with expect_error((ValueError, RuntimeError)):
tests/ttnn/unit_tests/operations/fused/parallel_sequential/test_parallel_sequential.py:802:
    def __init__(self, func, args, kwds):
>       self.gen = func(*args, **kwds)
E       TypeError: expect_error.<locals>.expect_error_() missing 1 required positional argument: 'message'
/usr/local/share/uv/cpython-3.10.19-linux-x86_64-gnu/lib/python3.10/contextlib.py:103: TypeError

FAILED tests/ttnn/unit_tests/operations/fused/parallel_sequential/test_parallel_sequential.py::TestMatmulFusion::test_fp32_mismatch_error - TypeError: expect_error.<locals>.expect_error_() missing 1 required positional argument: 'message'
```

**Run 2** (`53fd14fd7f49`):
```
__________________ TestMatmulFusion.test_fp32_mismatch_error ___________________
    def test_fp32_mismatch_error(self, device, expect_error):
        """MM(fp32=off) + LN(fp32=on) should raise."""
        ...
>       with expect_error((ValueError, RuntimeError)):
tests/ttnn/unit_tests/operations/fused/parallel_sequential/test_parallel_sequential.py:802:
    def __init__(self, func, args, kwds):
>       self.gen = func(*args, **kwds)
E       TypeError: expect_error.<locals>.expect_error_() missing 1 required positional argument: 'message'
/usr/local/share/uv/cpython-3.10.19-linux-x86_64-gnu/lib/python3.10/contextlib.py:103: TypeError

FAILED tests/ttnn/unit_tests/operations/fused/parallel_sequential/test_parallel_sequential.py::TestMatmulFusion::test_fp32_mismatch_error - TypeError: expect_error.<locals>.expect_error_() missing 1 required positional argument: 'message'
```

### Reproduction steps

```bash
pytest tests/ttnn/unit_tests/operations/fused/parallel_sequential/test_parallel_sequential.py::TestMatmulFusion::test_fp32_mismatch_error -xv
```

The job command (with simulator env) was:
```bash
pytest tests/ttnn/unit_tests/operations/fused -xv --splits 2 --group 1 --splitting-algorithm least_duration -m "not disable_fast_runtime_mode"
```
with `TT_METAL_SIMULATOR`, `TT_METAL_SLOW_DISPATCH_MODE=1`, and `TT_METAL_DISABLE_SFPLOADMACRO=1` set.

### Notes

The `expect_error` fixture (used as a context manager) changed its signature to require a `message` argument, but `test_fp32_mismatch_error` calls it with only an exception type tuple `(ValueError, RuntimeError)` and no `message`. This is a test/fixture API mismatch introduced between commits `25c2bfb980e8` and `53fd14fd7f49`.

**Determinism check:**
- **Run 1** terminal error: `TypeError: expect_error.<locals>.expect_error_() missing 1 required positional argument: 'message'` (line 3712 / 6003 of run1 log)
- **Run 2** terminal error: `TypeError: expect_error.<locals>.expect_error_() missing 1 required positional argument: 'message'` (line 3562 / 5449 of run2 log)
- Both runs fail with the same `TypeError` at the same call site (`test_parallel_sequential.py:802`).

<!-- AUTO-TRIAGE-METADATA-START -->
`Auto-triage-workflow: Sanity tests`
`Auto-triage-job-name: ttnn-sanity-tests / ttnn fused group 1 [sim_blackhole]`
<!-- AUTO-TRIAGE-METADATA-END -->

<!-- AUTO-TRIAGE-INFO-START -->
## About this issue (automated CI auto-triage)
This issue was opened and assigned by an automated AI system. It posts through a bot token, so notifications may look like they come from a person, but they are automated.

The owner is selected from recent commit history and job-ownership data. It reflects who is *likely able to fix or route* this, not necessarily who caused it, so it can be imperfect.

**If this was assigned to the wrong person, you can:**
- Comment `REASSIGN OWNER: <why this is outside your area / which team or test-area it belongs to>` and the system will automatically re-route it to a better-suited owner.
- Unassign yourself (this stops notifications to you).
- Close the issue if it is not a real or relevant failure.

The assigned owner is reminded about every 3 days until the issue is resolved. Full guide: https://tenstorrent.atlassian.net/wiki/spaces/MI6/pages/2545221692/Auto-Triage+Info
<!-- AUTO-TRIAGE-INFO-END -->

--- a/tests/ttnn/unit_tests/operations/fused/parallel_sequential/test_parallel_sequential.py
+++ b/tests/ttnn/unit_tests/operations/fused/parallel_sequential/test_parallel_sequential.py
@@ -799,7 +799,7 @@ class TestMatmulFusion:
             epsilon=1e-5,
         )
 
-        with expect_error((ValueError, RuntimeError)):
+        with expect_error((ValueError, RuntimeError), "fp32_dest_acc_en mismatch"):
             Sequential(mm, ln).build(device)
 
 

